src/docs/chunkcache.dox

```diff
@@ -27,4 +27,24 @@ Newer content is more likely to be accessed again, so we insert freshly-flushed
 This saves the chunkcache from having to read freshly-flushed data back from the object store soon after it was put there. When a tiered object is flushed and before it is deleted locally, we make an ingest call and the new or modified table will be inserted into the chunkcache.
 This function will also check for outdated pinned content in the chunk cache, ensuring the older versions of the pinned content can be cleaned up by chunk cache's eviction process.
 
+@section persistence Persisted content
+
+The chunk cache could contain a considerable quantity of content. This
+content would typically be on-disk, meaning it can be persisted across
+WiredTiger restarts. Doing this saves the user time (due to reduced latency
+when accessing data from disk, compared to networked block storage) and
+money (as these block storage accesses are often associated with a monetary
```

Comment:
- perhaps "resources" instead of "money"
- also you can have "as these block storage accesses are often associated with an API usage cost" don't need monetary as that is implied with the word cost.

Reply:
Fixed in cb4ead7 - thanks.